.trivyignore

```diff
@@ -0,0 +1,5 @@
+# org.scala-lang:scala-library (from upstream opensource dependency)
```

Comment:
Can we constraint it? Guessing yes given the age.

Reply:
This is kafka server image. Not kafka-streams application. We don't have any codebase and don't build the code. We just use apache tarball distro to build our image.

Reply:
got it.